IMAP synchronization  based on imapsync (#511)
NethServer/docs: administrator-manual/en/mail.rst

.. index::
   pair: email; filter

.. _email_imap_synchronization:

IMAP synchronization
====================

IMAP synchronization is based on an IMAP transfer tool called Imapsync.

> The purpose is to migrate an IMAP account emais from a remote server to the account
> of your email server. 

Reviewer: ```suggestion
The purpose is to migrate email messages from a remote IMAP account to a
local one.
```

> of your email server. The migration is recursive and incremental,
> you can run it as many times you want, the emails will be copied locally
> if they do not exist on the local server.

Reviewer: ```suggestion
of your email server. 

The migration is recursive and incremental and 
can be repeated as many times as needed. The emails will be copied locally
if they do not exist on the local server.
```

can be repeated as many times as needed. The emails will be copied locally
if they do not exist on the local server.

The system administrator of the local |product| does not need to know the

> password of the local user. However, he

Reviewer 2: Try to avoid male-only pronouns:

```suggestion
password of the local user. However, administrator
```

> password of the local user, the IMAP admin authentication is used. However, he

Reviewer: ```suggestion
password of the local user. However, he
```

> If you plan to copy email from a |product| to another |product|,
> the IMAP admin user is the account ``vmail``. The password can be found at
> :file:`/var/lib/nethserver/secrets/vmail`.

Reviewer: ```suggestion
If the remote IMAP server is also a |product|,
the IMAP admin user is ``vmail`` and its password can be read from
:file:`/var/lib/nethserver/secrets/vmail`.
```

> 
> In the IMAP synchronization panel you must use a specific username
> ``username@domain.com*vmail`` and the ``vmail`` password.

Reviewer: ```suggestion
The username with a ``*vmail`` suffix (e.g. ``username@domain.com*vmail``) and the ``vmail`` password has to be set in the IMAP synchronization panel.
```

> .. note:: The developer of Imapsync has written a page about servers supporting
>          the IMAP admin authentication, it is available `on his website 
>          <https://imapsync.lamiral.info/FAQ.d/FAQ.Admin_Authentication.txt>`_

Reviewer: ```suggestion
.. note:: 

    List of `IMAP servers with admin authentication <https://imapsync.lamiral.info/FAQ.d/FAQ.Admin_Authentication.txt>`_ in Imapsync documentation
```
